publishing/s3publisher.py

```diff
@@ -173,8 +174,9 @@ def publish_to_s3(directory, base_url, site_prefix, bucket, cache_control,
         else:
             start_time = datetime.now()
 
+            LOGGER.info(f'Deleting {file.s3_key}')
```

Comment:
LOGGER should precede the start_time assignment, unless you wanted the delta to reflect it as well.